### DISCUSSION TITLE
Is there a way to see how many days of recordings you have on calendar view?

### DISCUSSION THREAD

**Discussion by raspman** [blakeblackshear/frigate#15748] [Ask A Question] [answered]
I don't see a way to determine how many days each camera has recorded via the calendar view.  Is there documentation you could point me to on this?  Thanks

**Comment by NickM-27** [comment, marked as answer]
currently there is no direct way to do this, we could potentially add it in the future

**Reply by raspman** [reply]
That would be awesome - thank you.  Maybe a different color, like blue?
